webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/NodeConfigData.scala

```diff
@@ -1110,7 +1110,7 @@ class TestNodeConfiguration(
   lazy val clockVariables: Map[ComponentId, Variable] = {
     val spec = clockTechnique.getAllVariableSpecs.map(s => (s.name, s)).toMap
     Seq(
-      spec("CLOCK_FQDNNTP").toVariable(Seq("true")),
+      spec("CLOCK_FQDNNTP").toVariable(Seq("")), // testing mandatory value with default "true"
```

Comment:
this is the main test: all the written policy files are the same while we provide an empty value for the mandatory variable which as a default constrain.